apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.cs

```diff
@@ -785,7 +772,10 @@ private ExecuteSqlRequest GetExecuteSqlRequest()
             if (Parameters?.Count > 0)
             {
                 request.Params = new Struct();
-                Parameters.FillSpannerInternalValues(request.Params.Fields, request.ParamTypes);
+                // If SpannerConversionOptions is involved, build it from the connection.
+                // SpannerConversionOptions options = SpannerConversionOptions.ForConnection(SpannerConnection);
+                SpannerConversionOptions options = null;
```

Comment:
(same here)

Reply:
Changed the comment to refer to the other one :)